app/views/catalog/_universal_viewer_default.html.erb

```diff
@@ -0,0 +1,7 @@
+<iframe
+    src="<%= request&.base_url %>/universalviewer/uv/uv.html#?manifest=<%= document&.response&.response.dig(:docs,0,:iiif_manifest_url_ssi)  %>&config=examples-config.json"
```

Comment:
Still wondering about `iiif_manifest_url_ssi`.  The reason I never did this myself is because Spotlight doesn't generate manifests.

Reply:
:+1: if this is non-nil in the relevant context, maybe a unit test ensuring that it has the expected content is appropriate.

Reply:
@dunn @no-reply Here's the relevant test in spotlight:

https://github.com/projectblacklight/spotlight/blob/94b819926528a0331cd44d40f00be5349cc3a74e/spec/models/spotlight/resources/iiif_manifest_spec.rb#L72

I believe that Spotlight is generating the manifests here: https://github.com/projectblacklight/spotlight/blob/94b819926528a0331cd44d40f00be5349cc3a74e/app/presenters/spotlight/iiif_manifest_presenter.rb

Here's what one of them looks like: https://geniza-qa.curationexperts.com/spotlight/the-anna-s-c-blake-manual-training-school/catalog/9-33/manifest

Reply:
I'm super mystified, since it was always nil for me locally.  I don't have a login for the DCE sandbox; what does the JSON look like?

Reply:
Ha, this is getting weirder
```js
{
@context: "http://iiif.io/api/presentation/2/context.json",
@type: "sc:Manifest",
@id: "https://geniza-qa.curationexperts.com/spotlight/the-anna-s-c-blake-manual-training-school/catalog/9-32/manifest",
label: "Miss Anna S.C. Blake",
description: "oak ash pine rowan apple",
sequences: [
{
@type: "sc:Sequence",
@id: "https://geniza-qa.curationexperts.com/spotlight/the-anna-s-c-blake-manual-training-school/catalog/9-32/manifest/sequence/normal",
rendering: [ ],
canvases: [
{
@type: "sc:Canvas",
@id: "https://geniza-qa.curationexperts.com/spotlight/the-anna-s-c-blake-manual-training-school/catalog/9-32/manifest/canvas/9-32",
label: "Miss Anna S.C. Blake",
width: "326",
height: "400",
images: [
{
@type: "oa:Annotation",
motivation: "sc:painting",
resource: {
@type: "dctypes:Image",
@id: "9-32",
height: "400",
width: "326",
format: "image/jpeg",
service: {
@context: "http://iiif.io/api/image/2/context.json",
@id: "https://geniza-qa.curationexperts.com/images/25",
profile: "http://iiif.io/api/image/2/level2.json"
}
},
on: "https://geniza-qa.curationexperts.com/spotlight/the-anna-s-c-blake-manual-training-school/catalog/9-32/manifest/canvas/9-32"
}
]
}
]
}
]
}```

Reply:
And from the rails console, `#to_solr` has that as the value for that field?  My last guess is that it's an auth issue, since I can't see it without logging in.

Reply:
It's feeling more and more to me like a local unit test is called for.

Reply:
I'm not sure what I need to test, @dunn am I understanding that you thought that Spotlight 
didn't generate a IIIF manifest at all or just that this particular property had been `nil` for you in the past? Do I need to write a test that shows that Spotlight generates a manifest when you create an item?

Reply:
I'm slightly concerned by the amount of branching going on at `document&.response&.response.dig(:docs,0,:iiif_manifest_url_ssi)`. A view test showing the setup needed to achieve an expected result is probably a good starting place.

Reply:
Yeah, a test to show that the view has access to that Solr field sounds like a good idea.

Reply:
@dunn could you take another look at this?